packages/theme-data/README.md

```diff
@@ -51,6 +71,15 @@ console.log(redAccentedTheme);
 }
 ```
 
+## Available themes
+
+There are eight themes made up of four color schemes a two densities each. These themes are importable in three formats from respective folders in the build folder: [ESM](./build/esm), [JSON](./build/json), and [SCSS](./build/scss) 
+
+* Web light, medium (this is the default color scheme) and high density 
```

Comment:
I feel like we should put Web Light lower, as even though it is the current default, it is legacy and on the road to being deprecated.